x-pack/plugins/infra/public/pages/metrics/hosts/components/tabs/alerts_tab_badge.tsx

```diff
@@ -36,9 +36,12 @@ export const AlertsTabBadge = () => {
     );
   }
 
-  return (
+  const shouldRenderBadge =
+    typeof alertsCount?.activeAlertCount === 'number' && alertsCount.activeAlertCount > 0;
```

Comment:
just out of curiosity: under what circumstances `activeAlertCount` isn't a number?

Reply:
If `alertsCount` is not fetched yet it is `undefined`, and with the optional chain operator also `activeAlertCount` results to be `undefined`, which will make TS complain when trying to compare it with the 0 number